cmd/cloud-api-adaptor/main.go

```diff
@@ -37,7 +37,7 @@ func (cfg *daemonConfig) Setup() (cmd.Starter, error) {
 
 	if len(os.Args) < 2 {
 		fmt.Printf("%s <provider_name> [options] | version\n", os.Args[0])
-		fmt.Printf("   The <provider_name> should be one of aws|azure|ibmcloud|libvirt|vsphere>\n")
+		fmt.Printf("   The <provider_name> should be one of aws|azure|ibmcloud|ibmcloud-powervs|libvirt|vsphere>\n")
```

Comment:
Could you rebase this PR?

As https://github.com/confidential-containers/cloud-api-adaptor/pull/788 has been merged, we no longer need to hard-code cloud provider names in `main.go`.

Reply:
Addressed, PTAL.